mmv1/products/billing/ProjectInfo.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: ProjectInfo
+base_url: 'projects/{{project}}/billingInfo'
+create_verb: :PUT
```

Comment:
Is this resource updatable as well? If so, can you add a handwritten update test? If not, you can add immutable: true here

Reply:
It is updatable. 

I'll add the update test and update the PR

Reply:
Done